src/test_l1_tvSettings.c

```diff
@@ -2064,13 +2090,21 @@ void test_l1_tvSettings_negative_SaveBacklight (void)
 	UT_LOG("In:%s [%02d%03d]", __FUNCTION__,gTestGroup,gTestID);
 
 	tvError_t result = tvERROR_NONE ;
+	uint32_t videoSource = 0;
+	uint32_t pqValue = 0;
+	uint32_t videoFormat = 0;
+	uint32_t count = 0;
 	bool SupportAvailable = true;
+	char keyValue[UT_KVP_MAX_ELEMENT_SIZE] = { 0 };
 
+	videoSource = UT_KVP_PROFILE_GET_UINT32("tvSettings/VideoSource/index/0");
+	pqValue = UT_KVP_PROFILE_GET_UINT32("tvSettings/PictureMode/index/0");
+	videoFormat = UT_KVP_PROFILE_GET_UINT32("tvSettings/VideoFormat/index/0");
 	if (extendedEnumsSupported == true)
 	{
 		/* Step 01: Calling tvsettings SaveBacklight and expecting the API to return tvERROR_INVALID_STATE */
-		result = SaveBacklight((tvVideoSrcType_t)Configfile.videoSrcStruct.videoSourceValue[0], Configfile.picmodeStruct.pqValue[0],
-							   (tvVideoFormatType_t)Configfile.videoFormtStruct.videoFormatValue[0], 50);
+		result = SaveBacklight((tvVideoSrcType_t)videoSource, pqValue,
```

Comment:
SaveBacklight -> Where your using this function all cases need to be removed.

Your clearly not walking through this with the debugger, which is a requirement for doing any tests.

Reply:
Need further discussion to tackle it in different work element.

Reply:
Added  to #18 so that you can review as a requirement for cleanup.